src/ini/config.c

```diff
@@ -0,0 +1,79 @@
+#include "config.h"
+#include "ini.h"
+#include <string.h>
+
+int match(const char *section, const char *check_section, const char *name, const char *check_name) {
+        if (strcmp(section, check_section) == 0 && strcmp(name, check_name) == 0) {
```

Comment:
Please use ```strncmp()``` rather than ```strcmp()```.

Reply:
For that may I assume that the max str length is 1000?